moveit_cpp:
  planning_scene_monitor_options:
    name: planning_scene_monitor
    robot_description: robot_description
    joint_state_topic: /joint_states
    publish_planning_scene: true
    publish_geometry_updates: true
    publish_state_updates: true
    publish_transforms_updates: true

planning_pipelines:
  pipeline_names: ["ompl", "pilz_industrial_motion_planner", "chomp", "stomp"]

#plan_request_params:
#  planning_pipeline: ompl
#  max_velocity_scaling_factor: 0.5
#  max_acceleration_scaling_factor: 0.3
#  planning_attempts: 1

plan_request_params:
  planning_pipeline: pilz_industrial_motion_planner
  planner_id: PTP
  max_velocity_scaling_factor: 1.0
  max_acceleration_scaling_factor: 1.0
  planning_time: 0.5
  planning_attempts: 5

planning_scene_monitor_options:
  name: "planning_scene_monitor"
  robot_description: "robot_description"
  joint_state_topic: "/joint_states"
  attached_collision_object_topic: "/moveit_cpp/planning_scene_monitor"
  publish_planning_scene_topic: "/moveit_cpp/publish_planning_scene"
  monitored_planning_scene_topic: "/moveit_cpp/monitored_planning_scene"
  wait_for_initial_state_timeout: 10.0

ompl:
  planning_plugins: ["ompl_interface/OMPLPlanner"]
  request_adapters:
    - default_planning_request_adapters/ResolveConstraintFrames
    - default_planning_request_adapters/ValidateWorkspaceBounds
    - default_planning_request_adapters/CheckStartStateBounds
    - default_planning_request_adapters/CheckStartStateCollision
  response_adapters:
    - default_planning_response_adapters/AddTimeOptimalParameterization
    - default_planning_response_adapters/ValidateSolution
    - default_planning_response_adapters/DisplayMotionPath
  #start_state_max_bounds_error: 0.1

pilz_industrial_motion_planner:
  default_planner_config: LIN
  planner_configs:
    LIN:
      type: LIN
      velocity_scaling: 0.20
      acceleration_scaling: 0.20
